spec/services/search_identities_spec.rb

```diff
@@ -206,8 +206,8 @@
     end
 
     context 'cro_number search' do
-      context 'when query matches' do
```

Comment:
I would argue that you should have kept the existent test case to assert exactly that then.